tests/drivers/fprotect/negative/testcase.yaml

```diff
@@ -1,16 +1,24 @@
+common:
+  platform_allow:
+    - nrf9160dk/nrf9160
+    - nrf52dk/nrf52832
+    - nrf5340dk/nrf5340/cpuapp
+    - nrf52840dk/nrf52840
+  sysbuild: true
 tests:
   drivers.fprotect.negative:
-    sysbuild: true
-    platform_allow:
-      - nrf9160dk/nrf9160
-      - nrf52dk/nrf52832
-      - nrf5340dk/nrf5340/cpuapp
-      - nrf52840dk/nrf52840
-    integration_platforms:
```

Comment:
good to leave at least one integration platform (for the CI, where -G flag is used)